src/components/radio-group/radio-group.ts

```diff
@@ -17,15 +17,63 @@ import styles from './radio-group.styles';
 @customElement('sl-radio-group')
 export default class SlRadioGroup extends LitElement {
   static styles = styles;
+  private _value: string;
 
   @query('slot:not([name])') defaultSlot: HTMLSlotElement;
 
   /** The radio group label. Required for proper accessibility. Alternatively, you can use the label slot. */
   @property() label = '';
 
+  /** The current value of the radio group. */
+  @property()
+  get value() {
+    if (!this._value) return this.getCurrentValue();
+
+    return this._value;
+  }
+
+  set value(newValue) {
+    const index = this.getAllRadios().findIndex(el => el.value === newValue);
+    const oldValue = this._value;
+
+    if (index > -1) {
+      this.checkRadioByIndex(index);
+      this._value = newValue;
+      this.requestUpdate('value', oldValue);
```

Comment:
Documentation around this pattern said to call `requestUpdate` manually. I'm not sure this is needed, since we're not watching the value and I'm assuming it avoids re-triggering this setter.

Reply:
Request update will tell the component to update at the end of the current microtask. My thought was that `update()` is synchronous, allowing the rerender to occur so we don’t need to make this an async method.

Reply:
Cancel this. It would be better to poke the shadow root and set the value directly on the internal radio.